### ISSUE TITLE
No push on android when no chrome tab running

### ISSUE THREAD

**Issue by sjernigan** [guerrerocarlos/serverless-web-push-notifications#2]
Thanks for putting this up.  It helped me confirm a problem I'm seeing in my own app.  Namely, on Android, if you close all the browser tabs, you don't get any push notifications.  If you have chrome running even if it's in the background and on another page, it works.  I'm investigating if the message payload has anything to do with it.  Are you aware of this issue and have you seen a fix?

**Comment by sjernigan** [comment]
I should mention that this works as expected with Firefox.

**Comment by sjernigan** [comment]
I don't think this is particular to your code so I've described it in a [stack overflow](https://stackoverflow.com/questions/54915123/fcm-push-notifications-to-chrome-while-killed) and linked to your demo. Hope that is acceptable. Any thoughts?

**Comment by sjernigan** [comment]
Possible related [Chrome bug](https://bugs.chromium.org/p/chromium/issues/detail?id=935931&can=2&start=0&num=100&q=push%20notification%20android&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified&groupby=&sort=)

**Comment by sjernigan** [comment]
This issue seems to be resolved in the Chrome Dev 74 build.  In theory, you can close this issue after Apr 23.

**Comment by Eiim** [comment]
Currently running Chrome 75 on Android, seems to be fine
